### PR TITLE
feat: add test on AnswersOptions

### PR DESCRIPTION
**Pull request by HuziMuzi**
added tests on answer option

### CODE REVIEW

File: src/components/AnswersOptions/tests/AnswersOptions.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {fireEvent, render, screen} from '@testing-library/react-native';
+import {AnswersOptions} from '@src/components/AnswersOptions/index';
+
+const onPressMock = jest.fn();
+const AnswersOptionsProps = {
+  onPress: onPressMock,
+  data: ['one', 'two', 'thee'],
```
Comment (antoniosk10) [inline src/components/AnswersOptions/tests/AnswersOptions.test.tsx:7]: thee

File: src/components/AnswersOptions/tests/AnswersOptions.test.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+test('Should render view block, correct operation of btn', async () => {
```
Comment (antoniosk10) [inline src/components/AnswersOptions/tests/AnswersOptions.test.tsx:10]: btn

Comment (antoniosk10) [reply src/components/AnswersOptions/tests/AnswersOptions.test.tsx:10]: also try to describe test as concrete as possible. What does mean "correct operation"? I don't know condition of correctness

Comment (antoniosk10) [reply src/components/AnswersOptions/tests/AnswersOptions.test.tsx:10]: and add tests for styles

**Review by antoniosk10 — APPROVED** [review]

**Review by Julia-Kovalchuk — APPROVED** [review]

**Pull request merged**